pkg/slack/notifier.go

```diff
@@ -159,7 +158,7 @@ func (n *Notifier) notify(ctx context.Context, run *jobs.WorkflowRun) {
 	}
 
 	for _, channel := range channels {
-		if len(channel.conclusions) > 0 && !slices.Contains(channel.conclusions, run.Conclusion) {
+		if !channel.filter.Any(run) {
```

Comment:
Here should be channel model logic. Instead of accessing `channel.filter`, we should introduce a new method to `Channel` method to check if we should send a notification for this 'job'